pallets/loans/src/util.rs

```diff
@@ -57,7 +57,7 @@ impl<T: Config> DataProvider<T::PriceId, T::Rate> for NoPriceRegistry<T> {
 }
 
 impl<T: Config> DataFeeder<T::PriceId, T::Rate, T::AccountId> for NoPriceRegistry<T> {
-	fn feed_value(_: T::AccountId, _: T::PriceId, _: T::Rate) -> DispatchResult {
+	fn feed_value(_: Option<T::AccountId>, _: T::PriceId, _: T::Rate) -> DispatchResult {
```

Comment:
This is the issue related to `pallet-loans` benchmarks. Quite funny BTW: The `account()` benchmark method takes the `Account` type for deserializing from the destination of its usage. Because now it has changed to `Option<Account>`, it deserializes wrongly.

Reply:
Written down a tech-debt to myself: Now orml is updated this part can be simplified.

Reply:
Ah awesome, thanks for the catch @lemunozm 🙏